frontend/src/i18n/en.json

```diff
@@ -10,6 +10,21 @@
     "password": "Password",
     "title": "Sign in"
   },
-  "registration": "registration",
-  "text": "text"
+
+  "registration": {
```

Comment:
Használtájok az i18next-scanner-t, láthatólag nem azzal jött létre a fordítás, ugyanis hiányzik a magyar és a szerb.